AmplifyPlugins/API/AWSAPICategoryPluginIntegrationTests/REST/RESTWithUserPoolIntegrationTests/RESTWithUserPoolIntegrationTests.swift

```diff
@@ -101,14 +99,12 @@ class RESTWithUserPoolIntegrationTests: XCTestCase {
                     return
                 }
 
-                guard case .notAuthenticated = authError else {
-                    XCTFail("Error should be AuthError.notAuthenticated")
+                guard case .notAuthorized = authError else {
+                    XCTFail("Error should be AuthError.notAuthorized")
```

Comment:
i'll look into this further, need to figure out what API Plugin is returning for someone that isn't signed in, @royjit